db/schema.rb

```diff
@@ -1307,6 +1443,8 @@
     t.integer  "dod_bah"
     t.boolean  "preferred_provider",                       default: false
     t.boolean  "stem_indicator",                           default: false
+    t.string   "campus_type"
```

Comment:
these changes don't seem relevant to this PR

Reply:
bad merge possibly, will cleanup

Reply:
mixed database migrations on local, removed